platform/src/pages/account/creation/organisation/verify/[id]/create-org/details.jsx

```diff
@@ -447,15 +447,20 @@ const CreateOrganisationDetailsPageTwo = ({ handleComponentSwitch }) => {
 const CreateOrganisationDetailsPageThree = () => {
   const dispatch = useDispatch();
   const router = useRouter();
-  const gridLocationsState = useSelector((state) => state.grids.gridLocations);
-  const gridSitesLocations = gridLocationsState.map((grid) => grid.sites);
-  const gridLocationsData = [].concat(...gridSitesLocations);
+  // const gridLocationsState = useSelector((state) => state.grids.gridLocations);
+  // const gridSitesLocations = gridLocationsState.map((grid) => grid.sites);
```

Comment:
Just curious about this code you have commented out. Is it reserved for future usage or just for just? @FaithDaka